docs/what-is-zarf.md

```diff
@@ -0,0 +1,225 @@
+
+### (3) - Zarf publishes your package
+
+Once your package has landed, depending on what you've built into it, there are a couple of ways for the Zarf cluster to expose package contents:
+
+In the simplest usage scenario, your package consists of a single application (plus dependencies) and you configure the Zarf cluster to _serve your application directly to end users_. This mode of operation is called "Appliance Mode"&mdash; because it's small & self-contained like a kitchen appliance&mdash;and it is intended for use in environments where you want to run k8s-native tooling but need to keep a small footprint (i.e. single-purpose / constrained / "edge" environments).
+
+Find out more about what direct user service looks like in the "[Appliance Mode](#appliance-mode)" section.
+
+In the standard, more complex usage scenario, your package consists of updates for many apps / systems and you configure the Zarf cluster to _propagate updates to downstream systems_ rather than to serve users directly. This mode of operation is called "Utility Mode"&mdash;as it's main job is to _add utility_ to other clusters&mdash;and it is intended for use in places where you want to run independent, full-service production environments (your own [Big Bang](https://github.com/DoD-Platform-One/big-bang) cluster, perhaps?) but you need help tracking, caching & disseminating system / dependency updates.
+
+Find out more about what downstream cluster service looks like in the "[Utility Mode](#utility-mode)" section.
+
+&nbsp;
+
+
+## How it's used (from 1,000 ft)
+
+### Building a package
+
+<a href="../.images/what-is-zarf/build-the-package.png">
+  <img alt="how it works" src="../.images/what-is-zarf/build-the-package.png" height="290" />
+</a>
+
+#### (0) - Identify software to-be-moved
+
+Zarf doesn't build software&mdash;it helps you distribute software that already exists.
+
+Zarf can pull from lots of places like [Docker Hub](https://hub.docker.com/), [Iron Bank](./ironbank.md), [GitHub](https://github.com/), local filesystems, etc. but you have to make sure that Zarf has a clear path & appropriate access credentials. Be sure you know what you want pack & how to access it before you Zarf.
+
+Find out more about the types of software Zarf can move in the "[What can be packaged?](#what-can-be-packaged)" section.
+
+#### (1) - Preparation
+
+In order to build a Zarf package, you need to prepare:
+
+- a "packaging" workstation &mdash; which must have the `zarf` [CLI tool installed](./workstation.md#just-gimmie-zarf), and
+
+- a `zarf.yaml` file &mdash; which tells the Zarf what you need packaged.
+
+Find some detailed uses of the `zarf.yaml` file in [our examples](../examples/).
+
+#### (2) - Package
+
+Actually making a Zarf package out of a `zarf.yaml` file is a matter of calling a single, simple command: `zarf package`.  You'll see a `zarf-package-*.tar.zst` file pop into existence afterward&mdash;that's your package.
```

Comment:
technically this is `zarf package deploy` to see the menu or you can just do `zarf zarf.yaml` to skip the menu

Reply:
Wait, but this para here is about _making_ the package rather than deploying it, right?  So I guess the command should be `zarf package create`.

So you can run it both ways, right?  Either directly with `zarf package create` or with `zarf zarf.yaml` will both build the package?

I kinda prefer the one with `package` in it cuz that makes the CLI API a bit more explicit, which seems fitting for a "first touch"-ish file like this.

You think it'd read more clearly with just `zarf zarf.yaml` though?

Reply:
yeah updated me comment.  I agree `zarf package create` is better introduction, power users will like move to `zarf zarf.yaml`